benefits/enrollment/templates/enrollment/index.html

```diff
@@ -1,108 +1,113 @@
-                            form.submit();
-                        },
-                        onError: function (response) {
-                            /* This function executes when the
-                            /* server returns error or token is invalid.
-                            /* 400 or 500 will return. */
-                            amplitude.getInstance().logEvent(closedEvent, {status: "error", error: response});
+                      var form = $("form#{{ form_retry }}");
+                      form.submit();
+                  },
+                  onError: function (response) {
+                      /* This function executes when the
+                      /* server returns error or token is invalid.
+                      /* 400 or 500 will return. */
+                      amplitude.getInstance().logEvent(closedEvent, {status: "error", error: response});
 
-                            var form = $("form[action='{{ forms.tokenize_retry }}']");
-                            form.submit();
-                        },
-                        onCancel: function () {
-                            /* This function executes when the
-                            /* user cancels and closes the window
-                            /* and returns to home page. */
-                            amplitude.getInstance().logEvent(closedEvent, {status: "cancel"});
+                      var form = $("form#{{ form_retry }}");
+                      form.submit();
+                  },
+                  onCancel: function () {
+                      /* This function executes when the
+                      /* user cancels and closes the window
+                      /* and returns to home page. */
+                      amplitude.getInstance().logEvent(closedEvent, {status: "cancel"});
 
-                            return location.reload();
-                        }
-                    });
-                });
-            })
-                .fail(function(jqxhr, settings, exception) {
-                $(".loading").remove();
-                console.log(exception);
-            });
-    </script>
-  {% else %}
-    <h2>Error!</h2>
-    <p>No access token configured</p>
-  {% endif %}
+                      return location.reload();
+                  }
+              });
+          });
+      })
+      .fail(function(jqxhr, settings, exception) {
+        $(".loading").remove();
+        console.log(exception);
+      });
+  </script>
 
-  {% for f in page.forms %}
+  {% for f in forms %}
     {% include "core/includes/form.html" with form=f %}
   {% endfor %}
 {% endblock inner-content %}
 
 {% block call-to-action-button %}
-  {% if payment_processor %}
-    <button type="button" class="btn btn-primary btn-lg loading" role="status" disabled>
-      {{ payment_processor.loading_text }}
+  <button type="button" class="btn btn-primary btn-lg loading" role="status" disabled>{{ loading_text }}</button>
```

Comment:
I'm seeing something weird where this `loading_text` doesn't show up. The one on line 46 works though 🤔 

![benefits-enrollment-index](https://github.com/cal-itp/benefits/assets/25497886/61b1efdf-b177-4c4a-8cb9-2cac3df517b3)

Reply:
yeah I noticed this on mine too.